recipes/zoneinfo/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "backports.zoneinfo" %}
+{% set version = "0.2.1" %}
+{% set hash = "fadbfe37f74051d024037f223b8e001611eac868b5c5b06144ef4d8b799862f2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
+  patches:
+    - tzdata-location.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - tzdata
```

```suggestion
    - backports
    - python
    - pip
  run:
    - backports
    - python
    - tzdata
```
^-- Just to avoid file conflicts/clobbering when installing other `backports.*` packages.

---

The `backports` package just has a `backports/__init__.py` . If we add it in `host` we avoid `backports.zoneinfo` to include another one. And adding to `run` because we want to have one installed in any case.

---

^ I'm a bit confused by this one. Don't we want the specific `backports.zoneinfo` package?

---

All those `backports.*` packages make use of the Python "namespace package" `backports`. Which means each of them would install their own `backports/__init__.py` file if we don't prevent it. Installing multiple of those `backports.*` packages would then result in clobbering that `__init__.py` file. Which in turn would mean whenever you have multiple of those installed but remove only one of them, it would remove a file (i.e., `backports/__init__.py`) that is required by other packages that are still installed.
We have a separate Conda package `backports` which only consists of that single `backports/__init__.py` file. When we install that `backports` package, all other `backports.*` packages can share that same file without shipping their own, i.e., avoid the clobbering.

---

>  Don't we want the specific `backports.zoneinfo` package?

`backports.zoneinfo` also carries its `backports/zoneinfo/__init__.py`

---

Right, thanks for the explanation - makes sense!